test/test_view_ops.py

```diff
@@ -915,28 +915,37 @@ def _test_ravel(tensors, size, nc=False):
                 self.assertEqual(flat.shape, torch.Size([size]))
                 self.assertEqual(src.view(-1), flat)
                 self.assertEqual(flat._base, src)
```

Comment:
Should this be `self.assertIs(flat._base, src)`? We want to make sure that not only `flat._base == src` but `flat._base is src`, right?

Reply:
Oo, yes! The same reason goes in.